lib/checker.js

```diff
@@ -45,12 +45,62 @@ Checker.prototype.checkFile = function(path) {
 };
 
 /**
+ * Fixes single file.
+ *
+ * @param {String} path
+ * @returns {Promise * Errors}
+ */
+Checker.prototype.fixFile = function(path) {
+    if (!this._configuration.isFileExcluded(path)) {
+        return vowFs.read(path, 'utf8').then(function(data) {
+            var result = this.fixString(data, path);
+            return vowFs.write(path, result.output).then(function() {
+                return result.errors;
+            });
+        }, this);
+    }
+
+    return Vow.resolve(null);
+};
+
+/**
  * Checks directory recursively.
  *
  * @param {String} path
  * @returns {Promise * Error[]}
  */
 Checker.prototype.checkDirectory = function(path) {
+    return this._processDirectory(path, this.checkFile.bind(this));
+};
+
+/**
+ * Checks directory or file.
+ *
+ * @param {String} path
+ * @returns {Promise * Error[]}
```

Comment:
Non-blocker: I was wondering where this came from in https://github.com/jscs-dev/node-jscs/pull/1038/files#diff-605f23c7e3628b173a428e6a469e6ff6R52. 

Is that valid jsdoc?